hello-world-react-app/src/components/CounterComponent.tsx

```diff
@@ -0,0 +1,32 @@
+import { Component } from "react";
+
+export class CounterComponent extends Component {
+  state = {
+    count: 0,
+  };
+
+  increment() {
+    this.setState({
+      count: this.state.count + 1,
+    });
+  }
+
+  decrement() {
+    this.setState({
+      count: this.state.count - 1,
+    });
+  }
+
+  render() {
+    return (
+      <>
+        <h1>CounterComponent</h1>
+        {this.state.count}
+        <div>
+          <button onClick={this.increment.bind(this)}>increment</button>
```

Comment:
instead of writing bind all the time - for class methods use arrow functions

Reply:
It's a personal preference 🙂
We could have style guides for that but since it is not a production project it may be skipped